src/main/java/me/minidigger/hangar/controller/api/VersionsApiController.java

```diff
@@ -50,6 +59,7 @@ public VersionsApiController(ObjectMapper objectMapper, HttpServletRequest reque
 
     @Override
     public ResponseEntity<PaginatedVersionResult> listVersions(String pluginId, List<String> tags, Long limit, Long offset) {
+        apiVersionsDao.get().listVersions(pluginId, tags, false, limit, offset, null);
```

Comment:
The returned `List` has not been used here.

Reply:
oh yeah, I think that's because I was gonna finish working on the versions API later, ill add a todo for now.